playbooks/generate_workspace.yml

```diff
@@ -379,6 +380,13 @@
             egrep '(_kek)' |
               cut -d':' -f2
 
+    - name: Generate global/common secrets
+      ansible.builtin.set_fact:
+        secrets: "{{ secrets | default({}) | combine({item: lookup('password', '/dev/null chars=ascii_lowercase,ascii_uppercase,digits length=32')}) }}"
+      when: item not in secrets
+      loop:
+        - atmosphere_grafana_oauth_client_secret
+
```

Comment:
@mnaser do you agree on this way? There is no shared secret between roles still. For some shared/common vars, we used to set it as vars when call dependent roles in the role's meta so all are in one role's scope.
But i bet we will have this kind of common vars/secrets more in the future.

Reply:
I go down about this later, I think it's easier/cleaner to refactor things out into a role.